tut06/app.js

```diff
@@ -1,13 +1,13 @@
 var dir = process.argv[2],
     ext = process.argv[3],
-    listFiles = require('./listFiles');
+    filterFn = require('./solution_filter.js');
```

Comment:
There's no need to provide `.js` file extensions with commonJS require functions